app/models/user.rb

```diff
@@ -189,7 +189,7 @@ def preferred_language
   end
 
   def preferred_languages
-    @preferred_languages ||= Locale.list(languages)
+    Locale.list(languages)
```

Comment:
Rather than just getting rid of the memoization maybe we should use an `after_save` filter to set it to `nil` so that it will be recomputed on next read?